### DISCUSSION TITLE
tabs 自定义宽度的问题

### DISCUSSION THREAD

**Discussion by fbixiaoyu** [ant-design/ant-design-mobile#4069] [Q&A]
tabs 自定义宽度，分页右滑到底部的时候，会有一段空白。

**Comment by fbixiaoyu** [comment]
![image](https://user-images.githubusercontent.com/40888647/135099074-be7040f5-bd15-44aa-8e20-11552a055417.png)

**Reply by awmleer** [reply]
antd-mobile 的版本号是？
给个复现链接？
